src/elements/content-sidebar/activity-feed/task-new/_vars.scss

```diff
@@ -0,0 +1,8 @@
+@mixin avatar-badge-icon {
```

Comment:
consider renaming to `_variables` or `_mixins` (since there are no variables defined here). Also, to be safe we should import the main _variables.scss file (which has the colors defined). SASS will de-dupe any redundant imports, and that makes this file safe to import separately, should that case arise.

Reply:
Alternately, make the badge a component?

Reply:
Renamed to _mixins and added import of _variables. I could probably turn it into a reusable component (it's already a component, but it's not used in the top level avatar currently). However, I'd suggest doing that in a separate PR unless you object @alexkrolick.